Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
```

Comment:
@menslow looks like you did a `bundle update` which updated all the gems, generally not recommended since it can be hard to track down issues if something is suddenly broken -- so far using the site it seems like it's ok though. If this broke something then we could roll this back.